### DISCUSSION TITLE
Hide line if it contains?

### DISCUSSION THREAD

**Discussion by boelle** [WinMerge/winmerge#2078] [Q&A] [answered]
i'm editing a lot of subtitles and i have equal amount of lines that contains: -->

![image](https://github.com/WinMerge/winmerge/assets/3832239/0ee6c8e3-fa4a-46d9-9a99-a52da6624e5d)

is it possible to both ignore them but also hide them?

**Comment by sdottaka** [comment, marked as answer]
If you specify `-v -e "-->"` as an argument to the SelectLines plugin as shown below, I think you will get the results you expect.
![image](https://github.com/WinMerge/winmerge/assets/98126/0dc14991-d153-4013-8677-a17671dcd1a0)
If you just want to ignore the differences, you can use a line filter as shown below.
![image](https://github.com/WinMerge/winmerge/assets/98126/5b77ec44-6a56-46db-a336-58681347a969)

**Reply by boelle** [reply]
you are right, that made my progess much faster

instead of taking a month per movie subtitles i can now do maybe 3 movies worth in a day
